cetmix_tower_aws/readme/USAGE.md

```
@@ -0,0 +1,112 @@
+Please check the official Boto3 Documentation https://boto3.amazonaws.com/v1/documentation/api/latest/index.html for the detailed information about the services and methods provided by the Boto3 library.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)